PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/AnnotationReader.java

```diff
@@ -0,0 +1,211 @@
+package nl.tudelft.pl2016gr2.parser.controller;
+
+import nl.tudelft.pl2016gr2.model.Annotation;
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+public class AnnotationReader {
+
+  private static final Logger logger = Logger.getLogger(AnnotationReader.class.getName());
+
+  private final File file;
+
+  public AnnotationReader(File file) {
+    this.file = file;
+  }
+
+  @SuppressWarnings("checkstyle:MethodLength")
+  private Annotation readRow(Row row) {
+    Iterator<Cell> iterator = row.cellIterator();
+
+    Annotation annotation = new Annotation();
+
+    parseSpecimenID(annotation, iterator.next());
+    parseAge(annotation, iterator.next());
+    parseSex(annotation, iterator.next());
+    parseHIVStatus(annotation, iterator.next());
+    parseCohort(annotation, iterator.next());
+    parseDateOfCollection(annotation, iterator.next());
+    parseStudyGeographicDistrict(annotation, iterator.next());
+    parseSpecimenType(annotation, iterator.next());
+    parseMicroscopySmearStatus(annotation, iterator.next());
+    iterator.next(); // swallow specimenId column
+    parseDNAIsolation(annotation, iterator.next());
+    parsePhenotypicDSTPattern(annotation, iterator.next());
+    parseCapreomycin(annotation, iterator.next());
+    parseEthambutol(annotation, iterator.next());
+    parseEthionamide(annotation, iterator.next());
+    parseIsoniazid(annotation, iterator.next());
+    parseKanamycin(annotation, iterator.next());
+    iterator.next(); // swallow specimenId column
+    parsePyrazinamide(annotation, iterator.next());
+    parseOfloxacin(annotation, iterator.next());
+    parseRifampin(annotation, iterator.next());
+    parseStreptomycin(annotation, iterator.next());
+    parseDigitalSpoligotype(annotation, iterator.next());
+    parseLineage(annotation, iterator.next());
+    parseGenotypicDSTPattern(annotation, iterator.next());
+
+    return annotation;
+  }
+
+  private void parseSpecimenID(Annotation annotation, Cell cell) {
+    annotation.specimenID = cell.getStringCellValue();
+  }
+
+  private void parseAge(Annotation annotation, Cell cell) {
+    if (cell.getCellType() == Cell.CELL_TYPE_NUMERIC) {
+      annotation.age = new Double(cell.getNumericCellValue()).intValue();
```

Comment:
Is there any reason for not writing `(int)cell.getNumericCellValue()`?
This seems to do exactly the same, since the implementating of `intValue()` looks like this:
```java
public int intValue() {
    return (int)value;
}
```